airbyte-integrations/connectors/source-greenhouse/Dockerfile

```diff
@@ -12,5 +12,5 @@ COPY main.py ./
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.2.11
+LABEL io.airbyte.version=0.3.0
```

Comment:
What motivates this minor version bump?
It's not blocking, just wondering about your approach to this.
I'd guess it's because you consider an enriched schema is a new feature.

Reply:
yes, because of a schema change - I thought it must be a backwards incompatible change. Now I realize I should have bumped the major version, right? 
btw, can you prompt me the proper approach of rolling out this kind of change?

Reply:
I don't think it's a backward incompatible change, the acceptance tests did not fail. Schema validation error will still happen on users with previous version but it won't break their sync.